doc/content/getting-started/se-hands-on-guide/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "SENSOR EDITION HANDS ON TUTORIALS"
+description: ""
+weight: 3
+distributions: null
+---
+
+A guide for {{% gnse %}} hands-on videos and tutorials.
```

Comment:
This guide contains hands-on videos and tutorials for working with {{% gnse %}}.

Reply:
Done